study.md

```diff
@@ -50,13 +54,15 @@ Which ActiveRecord method creates new objects? Does this method persist objects
 as rows in the database, or is there another required method for persistence?
 
 ```md
-<!-- your response here -->
+The .create method creates new objects, however it does not commit it to the
+database. In order to save the object, we use the .save method after calling
+.create
 ```
```

Comment:
create has "an implied save" for lack of better wording.  For new you have to explicitly save.